dashboard/config/puma.rb

```diff
@@ -3,6 +3,7 @@
 require path
 
 if CDO.dashboard_sock
+  bind "tcp://#{CDO.dashboard_host}:9000"
   bind "unix://#{CDO.dashboard_sock}"
 else
   bind "tcp://#{CDO.dashboard_host}:#{CDO.dashboard_port}"
```

Comment:
We end up having a bit of repeated code here; would it be feasible for us to set `CDO.dashboard_port` to 9000, and replace this entire `if` block with something like

```ruby
  bind "unix://#{CDO.dashboard_sock}" if CDO.dashboard_sock
  bind "tcp://#{CDO.dashboard_host}:#{CDO.dashboard_port}"
```